### PR TITLE
Feature/title subtitle component

### PR DESCRIPTION
**Pull request by SinuheM**
This PR contain components that are used in different functionalities such as "Escoge tu perfil", "Aliados" and "Obras de arte"

### CODE REVIEW

**Review by VeryEvilTomato — APPROVED** [review]

**Review by dyepes62 — APPROVED** [review]

File: components/Subtitle.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default function Subtitle({ children }) {
+  return (
+    <h4 className="subtitle">
+      {children}
+    </h4>
+  );
+}
```
Comment (davidvm1) [inline components/Subtitle.js:7]: What is the criteria to define this subtitle component as a h4? Does it make sense based on the hierarchy?

File: components/Title.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default function Title({ children }) {
+  return (
+    <h3 className="title">
+      {children}
+    </h3>
+  );
+}
```
Comment (davidvm1) [inline components/Title.js:7]: What is the criteria to define this subtitle component as a h3? Does it make sense based on the hierarchy?

File: styles/globals.scss [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+@import "./base/typography";
+@import "./components/title";
+@import "./components/subtitle";
 
 body {
   background-color: var(--coral);
   margin: 0;
   font-family: var(--poppins);
-  font-size: 1.5em;
 }
```
Comment (davidvm1) [inline styles/globals.scss:11]: global.scss would be just the entry file with the imports, if you want some base styles for certain tags like the body, header, h1, etc you can put them in the base.scss file. Also keep in mind consistency, I see there is aheader h1 {...} in this file. IS better if you have in base styles for tags on a general level and if you need to style the h1 or h2 inside the header, better do it in the scss file of the header component

File: components/Subtitle.js (hunk 1 above)
Comment (SinuheM) [reply components/Subtitle.js:7]: This is based on the style guide page in Figma, there are styles for each typography

File: components/Title.js (hunk 2 above)
Comment (SinuheM) [reply components/Title.js:7]: This is based on the style guide page in Figma, there are styles for each typography

File: styles/globals.scss (hunk 3 above)
Comment (SinuheM) [reply styles/globals.scss:11]: Ty, then I'm going to clean the global.scss

**Pull request merged**